src/App.tsx

```diff
@@ -127,19 +128,8 @@ function App() {
             }}
           />
         </Route>
-        <Route path="/:language/Canada">
-          <TableauEmbed
-            url={CANADA_URLS}
-            key={`CanadianTableau${language}`}
-            desktopOptions={{
-              width: "80vw",
-              height: "4100px"
-            }}
-            mobileOptions={{
-              width: "90vw",
-              height: "2800px"
-            }}
-          />
+        <Route path="/:language/Partnerships/:name">
+          <Studies />
```

Comment:
nit: would it be a good idea to rename this component to match the page name?

Reply:
yea, will do 👍